lightning/src/ln/channelmanager.rs

```diff
@@ -7206,6 +7344,33 @@ where
 	}
 }
 
+impl Writeable for VecDeque<(Event, Option<EventCompletionAction>)> {
+	fn write<W: Writer>(&self, w: &mut W) -> Result<(), io::Error> {
+		(self.len() as u64).write(w)?;
+		for elem in self.iter() {
+			elem.write(w)?;
+		}
+		Ok(())
+	}
+}
+impl Readable for VecDeque<(Event, Option<EventCompletionAction>)> {
+	fn read<R: Read>(reader: &mut R) -> Result<Self, DecodeError> {
+		let len: u64 = Readable::read(reader)?;
+		const MAX_ALLOC_SIZE: u64 = 1024 * 16;
```

Comment:
```suggestion
		const MAX_ALLOC_SIZE: u64 = 1024 * 64;
```

Also this is declared in a bunch of places, could consolidate it